interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1268,6 +1565,9 @@ class liquid_vapor(phase):
     equations of state. The substance_flag parameter selects the fluid. See
     liquidvapor.cti and liquidvapor.py for the usage of this entry type.
     """
+    pure_fluids: dict[int, str]
```

Comment:
I think this needs a `ClassVar[]` wrapped around this, then you can assign it on this line too